projects/components/src/filtering/filter/parser/filter-parser-lookup.service.ts

```diff
@@ -28,6 +29,8 @@ export class FilterParserLookupService {
         return new ComparisonFilterParser();
       case FilterOperator.In:
         return new InFilterParser();
+      case FilterOperator.NotIn:
```

Comment:
This should be able to use the same parser as IN, no need for a new one (rename may be appropriate though)

Reply:
Updated!